src/Lykke.Service.PayInternal/Controllers/MerchantsController.cs

```diff
@@ -173,24 +172,22 @@ public class MerchantsController : Controller
 
                 await _merchantService.UpdateAsync(merchant);
             }
-            catch (InvalidRowKeyValueException ex)
+            catch (InvalidRowKeyValueException e)
             {
-                _log.WriteError(nameof(UpdateAsync), new { ex.Variable, ex.Value }, ex);
+                _log.Error(e, new
+                {
+                    e.Variable,
+                    e.Value
+                });
 
-                return BadRequest(ErrorResponse.Create(ex.Message));
+                return BadRequest(ErrorResponse.Create(e.Message));
             }
             catch (MerchantNotFoundException exception)
             {
-                _log.WriteWarning(nameof(UpdateAsync), request, exception.Message);
+                _log.Warning(exception.Message, context: request);
```

Comment:
Alexey Novicikhin usually suggests to save in context only data wich is used for searching by fields in kibana.